pkg/cluster/ignition/library.go

```diff
@@ -568,6 +579,46 @@ func OcneUser(sshKey string, sshKeyPath string, password string) (*igntypes.Conf
 	return ret, nil
 }
 
+// KubeadmPatches generates the set of files and links required to use the
+// kubeadm patch mechanism to allow for hybrid cluster component versions.
+func KubeadmPatches() (*igntypes.Config, error) {
```

Comment:
do these files have to get onto existing cluster nodes or just new ones?  If so, how do they get there since ignition is only used for new nodes.

Reply:
They are only on new nodes.  The patch files are only relevant for new nodes because they are consumed by "kubeadm init/join" and never used again.  However, the kubeadm commands do require that the patch directory actually exists.  This function makes the directory so that cluster nodes that are created with old images that do not have the patch files have an empty patch directory so that kubeadm invocations can succeed.